app/controllers/account_invitations_controller.rb

```diff
@@ -2,16 +2,16 @@ class AccountInvitationsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @account_invitations = current_user.account_invitations
+    @account_invitations = account.invitations
```

Comment:
can we follow the current pattern in the code and use helper method instead of @var ? 

@var is fragile - a type inside the name will probably lead to an error inside the code but it won'y be clear what the error is (no method error for nil says nothing)